src/Razor/src/Microsoft.VisualStudio.LiveShare.Razor/Guest/GuestProjectPathProvider.cs

```diff
@@ -101,7 +101,7 @@ internal virtual Uri GetHostProjectPath(ITextDocument textDocument)
         [MethodImpl(MethodImplOptions.NoInlining)]
         private string ResolveGuestPath(Uri hostProjectPath)
         {
-            return _liveShareSessionAccessor.Session?.ConvertSharedUriToLocalPath(hostProjectPath);
+            return _liveShareSessionAccessor.Session.ConvertSharedUriToLocalPath(hostProjectPath);
```

Comment:
Idea here is I'd rather fail fast if we have a misassumption than wait for another piece of the pipeline to fail.